packages/rich-text-editor/src/vaadin-rich-text-editor.js

```diff
@@ -350,36 +350,6 @@ class RichTextEditor extends RichTextEditorMixin(
         <div class="announcer" aria-live="polite"></div>
       </div>
 
-      <vaadin-confirm-dialog
-        id="linkDialog"
```

Comment:
I removed all the IDs from the dialog. They were not usable for referencing elements anymore, and the IDs of the slotted dialog content are in the light DOM, where we shouldn't have any IDs in general. Technically someone might have used these for targetting styles, the alternative would then be to target the slot attribute.